pandas/core/groupby/groupby.py

```diff
@@ -1265,9 +1265,40 @@ def count(self):
     @Appender(_doc_template)
     def mean(self, *args, **kwargs):
         """
-        Compute mean of groups, excluding missing values
+        Compute mean of the target column for groups that are defined by the groupby columns.
+        Missing values are excluded in computing the mean.
+        If there is one groupby column, the groups are the unique values in the specified groupby column.
+        If there are multiple groupby columns, the groups are the unique combinations of the specified groupby columns.
+
+        Returns
+        -------
+        pandas.core.series.Series
+            The average of the target column ('B' in the examples below)
+            grouped by the groupby columns ('A' and ['A', 'C'] in the examples below)
+
+        Example of groupby one column:the groups are stored as index in the result.
```

Comment:
make this a sentence.
Groubpy by one column. The result index are the group labels.